src/app/PageClient.tsx

```diff
@@ -30,7 +38,12 @@ export default function Home() {
         gridTemplateColumns={['100%', '100%', 'minmax(0, 0.6fr) minmax(0, 0.4fr)']}
       >
         <TxListTabs limit={DEFAULT_LIST_LIMIT_SMALL} />
-        <BlocksList limit={DEFAULT_BLOCKS_LIST_LIMIT} />
+
+        {activeNetworkKey.indexOf('naka') !== -1 ? (
```

Comment:
Should make this a function

Reply:
Makes sense. It's a temporary way to display the new list for testing, once Nakamoto is out, we'll entirely remove the old list and the forking logic.